packages/terra-form-textarea/docs/MigrationGuide.md

```diff
@@ -0,0 +1,19 @@
+# Migrating from terra-form's textarea Component to terra-form-textarea
+
```

Comment:
Additionally, lets remove this file. I just double checked with @mjhenkes & we'll include the migration guide in the release notes.

Reply:
Removed migration guide. https://github.com/cerner/terra-core/pull/982/commits/802777623b56e1099bdf699936a4de4e2853d6ff